yinjuchen/artsync
# fix: update package json and update test case

- update package.json to have the cy:open command (usage `npm run c:open`)
- update cypress to latest build
- add test cases to verify count of the elements
- Add nprmc for building the project via public npm url
